tests/functional/tests/test_ceph.py

```diff
@@ -121,22 +129,25 @@ def test_check_ceph_cluster_health_passed(self):
 
         self._remove_test_pool()
 
-    def test_check_ceph_cluster_health_failed(self):
-        """Test that shutdown of a single ceph-osd unit fails."""
+    def test_check_ceph_cluster_health_warning(self):
+        """Test warning result for shutdown of a single ceph-osd unit."""
         # juju-verify shutdown --units ceph-osd/1
         units = ["ceph-osd/0"]
         check = "shutdown"
         unit_objects = loop.run(find_units(self.model, units))
 
         self._add_test_pool()
-        self._wait_to_ceph_cluster(healthy=False)
+        self._wait_to_ceph_cluster("fail")
         # check that Ceph cluster is unhealthy
         verifier = get_verifier(unit_objects)
         result = verifier.verify(check)
-        logger.info("result: %s", result)
-        self.assertFalse(result.success)
+        logger.info("result test_check_ceph_cluster_health_failed: %s", result)
+        self.assertTrue(
+            any(partial.severity == Severity.WARN for partial in result.partials)
+        )
         self.assert_message_in_result(
-            r"\[FAIL\] ceph-mon\/\d: Ceph cluster is unhealthy", result
+            r"\[WARN\] ceph-mon\/\d: Ceph cluster is in a warning state",
+            result,
```

Comment:
I think this will fail because the severity level has been restored to FAIL (for the release of v1.0).

Reply:
Thanks, I miss this.